manufacturers/rules.stanza

```diff
@@ -3,6 +3,50 @@ defpackage ocdb/rules :
   import core
   import collections
   import jitx
+  import jitx/commands
+
+public defn min-copper-width () :
+    clearance(current-rules(),MinCopperWidth)
+public defn min-copper-copper () :
+    clearance(current-rules(),MinCopperCopperSpace)
+public defn min-copper-hole () :
+    clearance(current-rules(),MinCopperHoleSpace)
+public defn min-copper-edge () :
+    clearance(current-rules(),MinCopperEdgeSpace)
+public defn min-annular-ring () :
+    clearance(current-rules(),MinAnnularRing)
+public defn min-drill-diameter () :
+    clearance(current-rules(),MinDrillDiameter)
+public defn min-silkscreen-width () :
+    clearance(current-rules(),MinSilkscreenWidth)
+public defn min-pitch-leaded () :
+    clearance(current-rules(),MinPitchLeaded)
+public defn min-pitch-bga () :
+    clearance(current-rules(),MinPitchBGA)
+public defn max-board-width () :
+    clearance(current-rules(),MaxBoardWidth)
+public defn max-board-height () :
+    clearance(current-rules(),MaxBoardHeight)
+public defn min-silk-soldermask () :
+    clearance(current-rules(),MinSilkSolderMaskSpace)
+public defn min-silkscreen-text-height () :
+    clearance(current-rules(),MinSilkscreenTextHeight)
+public defn solder-mask-registration () :
+    clearance(current-rules(),SolderMaskRegistration)
+public defn min-th-pad-expand-outer () :
+    clearance(current-rules(),MinThPadExpandOuter)
+public defn min-soldermask-opening () :
+    clearance(current-rules(),MinSolderMaskOpening)
+public defn min-soldermask-bridge () :
+    clearance(current-rules(),MinSolderMaskBridge)
+public defn min-hole-hole () :
+    clearance(current-rules(),MinHoleToHole)
+public defn min-pth-pin-solder () :
+    clearance(current-rules(),MinPthPinSolderClearance)
+
+public defn print-current-rules () :
+  println("min-copper-width %_ \nmin-copper-copper %_ \nmin-copper-hole %_ \nmin-copper-edge %_ \nmin-annular-ring %_ \nmin-drill-diameter %_ \nmin-silkscreen-width %_ \nmin-pitch-leaded %_ \nmin-pitch-bga %_ \nmax-board-width %_ \nmax-board-height %_ \nmin-silk-soldermask %_ \nmin-silkscreen-text-height %_ \nsolder-mask-registration %_ \nmin-th-pad-expand-outer %_ \nmin-soldermask-opening %_ \nmin-soldermask-bridge %_ \nmin-hole-hole %_ \nmin-pth-pin-solder %_" % 
+    [min-copper-width() min-copper-copper() min-copper-hole() min-copper-edge() min-annular-ring() min-drill-diameter() min-silkscreen-width() min-pitch-leaded() min-pitch-bga() max-board-width() max-board-height() min-silk-soldermask() min-silkscreen-text-height() solder-mask-registration() min-th-pad-expand-outer() min-soldermask-opening() min-soldermask-bridge() min-hole-hole() min-pth-pin-solder()])
```

Comment:
You can do this : 
```
#for rule in `[min-copper-width, 
               min-copper-copper, 
                    ... ] : 
  println("%_: %_", [`rule, rule()])
```
which will spit them onto separate lines and also be extensible when we add or remove new rules

Reply:
This would work for most rules, but for some of them, I shortened the names. For example:
```
public defn min-copper-copper () :
    clearance(current-rules(),MinCopperCopperSpace)
```